src/setup.ts

```diff
@@ -6,11 +6,12 @@ import { typeDefs } from './schema';
 import { resolvers } from './resolver';
 
 export const startServer = async () => {
-  await createConnection({
+  const connection = await createConnection({
     type: 'postgres',
     url: process.env.DATABASE_URL,
     entities: ['src/entity/**/*.ts'],
   });
+  connection.synchronize();
```

Comment:
Se for fazer um synchronize a cada connection, vc pode tbm mandar um `synchronize: true` no próprio createConnection ali em cima.